stdlib/public/core/VarArgs.swift

```diff
@@ -502,7 +502,11 @@ final internal class _VaListBuilder {
     appendWords(arg._cVarArgEncoding)
   }
 
-  @_inlineable // FIXME(sil-serialize-all)
+  // NB: This function *cannot* be @inlinable because it expects to project
+  // and escape the physical storage of `_VaListBuilder.alignedStorageForEmptyVaLists`.
+  // Marking it inlinable will cause it to resiliently use accessors to
+  // project `_VaListBuilder.alignedStorageForEmptyVaLists` as a computed
+  // property.
```

Comment:
Same here. We're supposed to fold all the va_list-building down to primitive ops.